packages/api/README.md

```diff
@@ -21,6 +21,15 @@ workers_dev = true
 account_id = "<what does the `wrangler whoami` say>"
 ```
 
+- `npm run build` - Build the bundle
```

Comment:
Did you need to build the bundle manually? `wrangler dev|publish` should be running the build command as defined in the wrangler.toml.

Reply:
Yes because first time round when you try to set secrets it'll automatically try and create a worker for you and needs a built bundle.